int16 positionx 
int16 positiony 
int16 orientation 

int16 value 
int8 robotStep 
int8 pass 
int8 ball_detect 
int8 ir_detect 
int16 ball_positionx 
int16 ball_positiony 

int16 targetfsmx 
int16 targetfsmy 

int16 obs1x 
int16 obs1y 
int16 obs1Detect 
int16 obs2x 
int16 obs2y 
int16 obs2Detect 
int16 obs3x 
int16 obs3y 
int16 obs3Detect 
int16 obs4x 
int16 obs4y 
int16 obs4Detect 

float64 ball_speedX 
float64 ball_speedY 
int16 ballTheta 

int8 doneshoot 

